iceoryx2-cal/src/zero_copy_connection/common.rs

```diff
@@ -728,11 +728,14 @@ pub mod details {
                 AdaptiveWaitBuilder::new()
                     .create()
                     .unwrap()
-                    .wait_while(|| {
-                        self.storage.get().channels[channel_id.value()]
-                            .submission_queue
-                            .is_full()
-                    })
+                    .timed_wait_while(
```

Comment:
Please revert this. This breaks the contract of this function, it is called `blocking_send` and shall block without a timeout.

Why did you adjust this?